CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 2.8)
+
+include(GNUInstallDirs)
+
+find_package(LLVM REQUIRED CONFIG)
+message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
+message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
+
+add_definitions(${LLVM_DEFINITIONS})
+include_directories(${LLVM_INCLUDE_DIRS})
+link_directories(${LLVM_LIBRARY_DIRS})
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
```

Comment:
We much have to copy what llvm does. If it is built without rtti, we cannot choose, otherwise we won't link together (vtables missing etc.). See REQUIRES_RTTI in autoconf/configure.ac. There is if (llvm <= 2.6) disable rtti btw :).